src/harness/testcases/testdata/gwpz_record_0.json

```diff
@@ -0,0 +1,58 @@
+{"record":
+{
+  "id": "incumbent/wisp/0",
+  "type": "wisp",
+  "deploymentParam": 
+    {
+      "installationParam": {
+        "longitude": -99.7,
+        "latitude": 39.1,
+        "antennaBeamwidth": 30,
+        "antennaAzimuth": 270,
+        "antennaDowntilt": -90
+      },
+      "operationParam": {
+        "operationFrequencyRange": {
+          "lowFrequency": 3650000000.0,
+          "highFrequency": 3700000000.0
+        }
+      }
+    }
+},
+"zone":{
```

Comment:
I think you can actually leave out the zone here since the intent is to inject a GWBL, not a GWPZ (see the documentation [here](https://docs.google.com/document/d/19YbeQsSzm5kCIOeVzsF8piqI0Lizy3p345aiVGf-Hlg/edit?pli=1#heading=h.rz2qbup2aut9) which someone else helpfully reminded me of on Tuesday; sorry for the back-and-forth on this, it's completely my fault).